lib/mirage_impl_http.mli

```diff
@@ -1,9 +1,19 @@
+open Functoria
+
 type http
 
-val http : http Functoria.typ
+val http : http typ
+
+val cohttp_server : Mirage_impl_conduit.conduit impl -> http impl
+
+val httpaf_server : Mirage_impl_conduit.conduit impl -> http impl
```

Comment:
I'm not sure that it's relevant to continue to support it when we know that `Httpaf_mirage` has some bugs (/cc @TheLortex) and when `paf` exists and it is used. At least, can you delete the `httpaf` support? I will come back with the `paf` support.